patterns/1-initial/principles.md

```diff
@@ -0,0 +1,147 @@
+## Title
+
+Explicit InnerSource Principles
+
+## Patlet
+
+InnerSource as a term is circulating among employees. The usual explanation of
+"applying Open Source best practices inside an organisation" does not work well
+though with people lacking an Open Source background. As a remedy the most
+important principles of InnerSource get published widely.
+
+## Problem
+
+The organisation is trying to role out InnerSource at a larger scale. The
+initiative started among Open Source enthusiasts. The goal is now to get buy in
+from people that are lacking Open Source experience. As a result the typical
+slogan of "applying Open Source best practices" is not longer sufficient to
+transport the message of what InnerSource is, which problems it solves and which
+tools it uses for solving these issues.
+
+## Context
+
+In an organisation early experiments have shown that open source collaboration
+best practices can be beneficial. The goal now is to move the initiative to
+teams and individuals lacking a deep background in open source.
+
+The goal now is to clearly communicate the goals of the InnerSource initiative
+as well as a clear path towards achieving these goals.
+
+## Forces
+
+Teams have trouble communicating exactly what the important aspects of
```

Comment:
```suggestion
* Teams have trouble communicating exactly what the important aspects of
```

I suggest to use bullet lists here for the distinct forces, so that readers can check more easily if these apply for them too.